dispatchers_test.go

```diff
@@ -679,7 +803,7 @@ func TestDispatchRunFlags(t *testing.T) {
 	args := []string{"echo \"stuff\""}
 	original := "RUN --mount=type=bind,target=/foo echo \"stuff\""
 
-	if err := run(&mybuilder, args, nil, flags, original); err != nil {
+	if err := run(&mybuilder, args, nil, flags, original, nil); err != nil {
 		t.Errorf("dispatchAdd error: %v", err)
```

Comment:
```suggestion
		t.Errorf("run error: %v", err)
```

Reply:
Please do a pass checking that `Errorf` messages match the function that produced the assorted errors.  I'm only easily able to comment on lines that this PR touches.